homeassistant/components/govee_ble/sensor.py

```diff
@@ -0,0 +1,149 @@
+"""Support for govee ble sensors."""
+from __future__ import annotations
+
+from typing import Optional, Union
+
+from govee_ble import DeviceClass, DeviceKey, SensorDeviceInfo, SensorUpdate, Units
+
+from homeassistant import config_entries
+from homeassistant.components.bluetooth.passive_update_coordinator import (
+    PassiveBluetoothCoordinatorEntity,
+    PassiveBluetoothDataUpdate,
+    PassiveBluetoothDataUpdateCoordinator,
+    PassiveBluetoothEntityKey,
+)
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import (
+    ATTR_MANUFACTURER,
+    ATTR_MODEL,
+    ATTR_NAME,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo, EntityDescription
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+
+SENSOR_DESCRIPTIONS = {
+    (DeviceClass.TEMPERATURE, Units.TEMP_CELSIUS): SensorEntityDescription(
+        key=f"{DeviceClass.TEMPERATURE}_{Units.TEMP_CELSIUS}",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=TEMP_CELSIUS,
```

Comment:
This could be

```
DESCRIPTIONS = {
    (DeviceClass.MOTION, None): BinarySensorEntityDescription(
        key=f"{DeviceClass.MOTION}",
        device_class=BinarySensorDeviceClass.MOTION,
    ),
......
```